modules/ebi-metagenomics/summarisegoslims/main.nf

```diff
@@ -0,0 +1,52 @@
+
+process SUMMARISEGOSLIMS {
```

Comment:
should it be summarisEDgoslims or summariZegoslims? (my english is far from ideal, ignore me if I'm saying rubbish)

Reply:
summarise is british and summarize american, so both are correct 😄